src/site/content/en/fast/image-cdns/index.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: Use image CDNs to optimize images
+authors:
+  - katiehempenius
+description: |
+  Image CDNs are excellent at optimizing images. 
+  Switching to an image CDN can yield a 40-80% savings in image bytes.
+date: 2019-07-28
+codelabs:
+  - codelab-cloudinary-url-api
+---
+
+## Why use an image CDN?
+
+Image CDNs are excellent at optimizing images. Switching to an image CDN can yield a [40-80% savings](https://www.youtube.com/watch?v=YJGCZCaIZkQ&t=1010s) in image bytes. In theory, it is possible to achieve the same results by using only build scripts - but in practice this is rare.
+
+
+## What is an image CDN?
+
+Image CDNs specialize in the transformation, optimization, and delivery of images. You can also think of them as an API for accessing and manipulating the images used on your site. For images loaded from an image CDN, an image URL indicates not only which image to load, but also parameters like size, format, and quality. This makes it easy to create different variations of an image for different use cases.
+
+
+
+<figure class="w-figure  w-figure--center">
+  <img src="./image-cdn-requests.jpg" alt="Shows the requests/responses to/from an image CDN">
+</figure>
+
+
+Image CDNs are different from build-time image optimization scripts in that they create new versions of images as they are needed. As a result, they are generally better suited than build scripts for creating images that are heavily customized for each individual client.
+
+
+## Image URLs
+
+Image URLs used by image CDNs convey important information about an image and the transformations and optimizations that should be applied to it. URL formats will vary depending on image CDN, but at a high-level, they all have similar features.
+
+
+
+<figure class="w-figure  w-figure--center">
+  <img src="./image-cdn-url.jpg" alt="Example of a image CDN image URL">
+</figure>
+
+
+
+### Origin
+
+An image CDN can live on your own domain or the domain of your image CDN. Third-party image CDNs typically offer the option of using a custom domain (for example, yours) for a fee. Using your own domain makes it easier to switch image CDNs at a later date because no URL changes will be needed.
+
+Figure 2 uses the image CDN’s domain (“example-cdn.com”) with a personalized subdomain, rather than a custom domain.
```

Comment:
```suggestion
Figure 2 uses the image CDN’s domain ("example-cdn.com") with a personalized subdomain, rather than a custom domain.
```

I think we have a policy of using straight quotes just to avoid folks accidentally copy/pasting a smart quote into their text editor. https://web.dev/handbook/grammar/#quotation-marks-and-apostrophes